### ISSUE TITLE
RF PT ignores parameter weight decay, only uses the global optimizer setting

### ISSUE THREAD

**Issue by albertz** [rwth-i6/returnn#1439]
`rf.Parameter.weight_decay` is ignored in the PyTorch engine.
